packages/rocketchat-language-plugin/package.js

```diff
@@ -0,0 +1,25 @@
+Package.describe({
+	name: 'rocketchat:language-plugin',
+	version: '0.0.1',
+	summary: 'language plugin for Rocket.Chat'
+});
+
+Package.onUse(function(api) {
+	api.use('accounts-base');
+	api.use('ecmascript');
+	api.use('rocketchat:lib');
+	api.use('rocketchat:ui-message');
+	api.use('templating');
+
+	api.add_files('server/server.js', 'server');
+	api.add_files('server/add_or_update_user.js', 'server');
+	api.add_files('server/leave_automatic_channel.js', 'server');
```

Comment:
Why are you putting every function into its own file? Put all which you use together.